src/YAML.php

```diff
@@ -38,7 +38,7 @@ class YAML
      */
     public function __construct($In = '')
     {
-        if ($In) {
+        if (isset($In)) {
```

Comment:
`$In` should already always be set anyway, due to that it's one of method's parameters. In this case, simply checking truthiness would be better, because when instantiating the object, if the parameter is omitted (which is okay to do, because it has a default value; an empty string), `$In` will still exist (i.e., will be set), but will be an empty string (so, will evaluate as false).